src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
+your GitHub, GitLab, or Bitbucket account (cloud versions only, although
+CodeStream does have integrations with the on-prem versions once you've
+signed up). If you're a New Relic customer, you can also sign up with a user api
+key from your New Relic One account.
 
-Signing up via your code host also connects your repositories to CodeStream. If the email address you're using with your code host isn't your work email, you should create CodeStream-specific credentials instead.
+If the email address you're using with your code host isn't your work email, you
+should sign up with your work email address instead.
 
-If you sign up via CodeStream, your next step will be to confirm your email address by entering a code sent to you via email. You can paste the code into any of the boxes rather than typing each number individually.
+If you sign up using your email address, your next step will be to confirm the
+email address by entering a code sent to you via email. You can paste the code
+into any of the boxes rather than typing each number individually.
 
 ![Confirm Email](./images/EmailConfirmation1.png)
 
-## CodeStream organizations [#organizations]
-
-A CodeStream organization is where you and your teammates will discuss code. Similar to a Slack workspace, all of the developers in your company are in the same CodeStream organization. CodeStream's activity feed keeps things relevant for you by showing activity related to the code you have open in your IDE. 
-
-The discussions about code build up a knowledge base that is a company-wide resource, so the only reason to have multiple organizations on CodeStream is if you truly need separation. For example, you might have an organization for your day job and another for an open-source project you work on. Or maybe you're a consultant that's a member of different CodeStream organizations for each of your clients.
-
 ## Create or join an organization [#create-org]
 
-If you're invited to join an organization on CodeStream, sign up with the same email address the invitation was sent to. You'll automatically be added to that organization.
+A CodeStream organization is where you and your teammates will discuss code.
+Similar to a Slack workspace, all of the developers in your company can be in
+the same CodeStream organization. 
 
-If you're the first person from your company to sign up for CodeStream you can go ahead and create a new organization. Otherwise, there may be existing CodeStream organizations available for you to join based on your email domain. If you think your company is already on CodeStream, but don't see an organization to join, make sure that you've signed up with your work email.
+If you're invited to join an organization on CodeStream, sign up with the same
+email address the invitation was sent to and you'll automatically be added to
+that organization. Alternatively, there may be existing CodeStream organizations
+available for you to join based on your email domain.
```

Comment:
I don't quite understand the distinction that's being made here. Isn't the "alternatively" here just restating what you're saying in the previous paragraph or is there something different happening here?

Reply:
No, the previous paragraph simply states what an org is, and that your company only needs one. The "alternatively" line is about the fact that you may be able to join that org based on your email domain... but that is not a guaranteed thing. Only happens if the admin of that org has turned on domain-based joining. Just like with Slack.

Reply:
I think there's a funny conflation in this doc between admins and users of CodeStream. It would be good to clarify what admins can do and what users can do. I think the lines are somewhat blurred at the moment.